packages/rum/test/viewCollection.spec.ts

```diff
@@ -85,6 +85,23 @@ function mockHash(location: Partial<Location>) {
   }
 }
 
+function mockHTMLElement() {
+  spyOn(document, 'getElementById').and.callFake((elementId: string) => {
+    if (elementId === 'testHashValue') {
+      const dummyElement = document.createElement('div')
+      dummyElement.id = 'testHashValue'
+      const linkELement = document.createElement('a')
+      linkELement.href = '#testHashValue'
+      dummyElement.appendChild(linkELement)
+
+      return dummyElement
+    }
+
+    // tslint:disable-next-line: no-null-keyword
+    return null
```

Comment:
```suggestion
    return elementId === 'testHashValue'
```

Reply:
the jasmine Mock expect a `HTMLElement | null` result so `return elementId === 'testHashValue'` would not work here

Reply:
do we care?

Reply:
From my understanding `spyOn(document, 'getElementById').and.callFake` does

Reply:
Since we don't need the element that you create, I would rather go with something like `(elementId === 'testHashValue' as unknown) as any`

Reply:
Ok I thought that the cost of mocking a real HTMLElement was worth it but I will implement your version